lib/shoryuken/polling/base.rb

```diff
@@ -60,7 +60,11 @@ def ==(other)
       private
 
       def delay
-        Shoryuken.options[:delay].to_f
+        if name == 'default'
+          Shoryuken.options[:delay].to_f
+        else # allow delay settings per group
+          Shoryuken.options[:groups].to_h[name]).to_h[:delay]
```

Comment:
@lucas-ibotta you probably need `.to_f` as well for defaulting to `0.0`, like the main one.

Reply:
Maybe:

```ruby
Shoryuken.options[:groups].to_h[name].to_h.fetch(:delay, Shoryuken.options[:delay]).to_f
```

Reply:
Done. Much cleaner. That code does cause one new code climate failure though.
I'm unsure about the Travis failures too. The tests pass for me locally.

Reply:
```
opts = Shoryuken.options
opts[:groups].to_h[name].to_h.fetch(:delay, opts[:delay]).to_f
```
That might pass code climate?